admin/views/privacy.ejs

```diff
@@ -0,0 +1,127 @@
+<div class="grid-row">
+    <div class="column-two-thirds">
+        <%- partial('partials/_breadcrumbs', { breadcrumbs: breadcrumbs }) %>
+    </div>
+    <div class="column-one-third">
+        <%- partial('partials/_name_and_sign_out') %>
+    </div>
+</div>
+
+<div class="grid-row">
+    <div class="column-two-thirds">
+        <h1 class="heading-xlarge">Privacy Notice</h1>
+        <ul class="list list-contents">
+            <li class="list-item--dashed"><a href="#overview">Overview</a></li>
+            <li class="list-item--dashed"><a href="#data-collection-and-access">What data will be collected and who can access it?</a></li>
+            <li class="list-item--dashed"><a href="#retraining-personal-data">Retaining personal data</a></li>
+            <li class="list-item--dashed"><a href="#further-information">Further information</a></li>
+        </ul>
+        <hr>
+        <div id="overview">
+            <h2 class="heading-large">Overview</h2>
+            The Scottish Qualification Authority (SQA) is administering the multiplication tables check (MTC) technical
+            pre-test trials on behalf of the Standards and Testing Agency (STA).
+            The trials will take place in a sample of schools from 25 June to 6 July 2018.
+            <br/><br/>
```

Comment:
There are many unexplained uses of `<br/>` tags - apparently for spacing.  I'd suggest removing them and using `<p>` tags instead.  Additionally, the first para should have the `lede` class and appear slightly larger than usual.

Reply:
@jon-shipley replaced all br tags with paragraph tags on each paragraph.
Lede is avoided as per our discussion with Sinead